### ISSUE TITLE
A11y_A11y_.NETOnlineREPL_Benchmark_.NET Editor_Screen reader: Screen readers does not give more information for the suggestion in the editor.

### ISSUE THREAD

**Issue by Charanrajkb** [dotnet/try#790]
**Environment details:**
https://docs.microsoft.com/
Browser: Chromium Edge Version 80.0.361.50 (Official build) (64-bit)
OS Details:Windows 10 

GitHubTags:#A11yTCS;#A11ySev2;#A11y_.NETOnlineREPL_Benchmark;#.NET Online REPL;#Benchmark;#DesktopWeb;#Win10;#MAS1.3.1;#Accessibility;#.NET Online REPL;#Narrator;#NVDA;#chromiumEdge;#Chrome;#A11yMAS;

**​Repro Steps:​**
1.Hit the URL "https://docs.microsoft.com/" and login with appropriate credentials.
2.Tab till "Enter focus Mode" button and press enter.
3.Start narrator and navigate till the editor and type any alphabets.
4.Observe the narrator readings.

**Actual Result:​**
In the editor section when type aa alphabet there are suggestion but the narrator does not announce the suggestion are updated and it does not give the information to navigate inside the suggestion.

**Expected Result:​**
Narrator must give more instruction to navigate inside the suggestion using Down/Up arrow.

**User Impact:​**
Screen readers users will not understand the suggestions are present they don not understand how to navigate inside all suggestions.

**Attachment:**
[790_A11y_.NETOnlineREPL_.NET Editor_Screen reader.pptx](https://github.com/dotnet/try/files/4207905/790_A11y_.NETOnlineREPL_.NET.Editor_Screen.reader.pptx)

**Comment by jonsequitur** [comment]
Closing as external. This is a Monaco issue.

**Comment by keerthiduvvuri** [comment]
#Closed;
